src/domain/bitcoin/lightning/index.ts

```diff
@@ -18,3 +20,12 @@ export const PaymentSendStatus = {
   Pending: { value: "pending" },
   AlreadyPaid: { value: "already_paid" },
 } as const
+
+const pubkeyRegex = /^[a-f0-9]{66}$/i
+export const checkedToPubkey = (pubkey: string): Pubkey | ValidationError => {
+  if (pubkey.match(pubkeyRegex)) {
+    return pubkey as Pubkey
+  } else {
+    return new ValidationError("Pubkey conversion error")
```

Comment:
Aren't we using `ValidationError` only as a base class... shouldn't this be `class PubKeyConversionError extends ValidationError`?

Reply:
you're right. fixed